package.json

```diff
@@ -30,9 +30,10 @@
     "graphql": "^14.0.2",
     "graphql-tag": "^2.10.0",
     "marked": "^0.5.1",
-    "react": "^16.8.2",
+    "react": "^16.9.0-alpha.0",
```

Comment:
This is for avoiding a ton of warnings about needing to use `act` https://github.com/testing-library/react-testing-library/issues/281